dnf-docker-test/features/steps/test_behave.py

```diff
@@ -189,7 +202,7 @@ def then_package_state(context, std_message, state, line_start):
     elif std_message == 'stderr':
         message = context.cmd_error.split('\n')
     else:
-        raise AssertionError('The std_message {} is not allowed option for Then statement (allowed stdout, stderr)'
```

Comment:
it uses `repr()` to quote variable. so if somehow you have `"` in variable it will not be shown as `"a"b"` and it will be shown as `'a"b'`. and some other cases